compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -1360,67 +1340,85 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
 
         let opaque_ty_def_id = self.local_def_id(opaque_ty_node_id);
 
-        let mut collected_lifetimes = FxHashMap::default();
+        let mut collected_lifetimes = Vec::new();
```

Comment:
Comment! What goes in this vector?

```suggestion
        // Contains the new lifetime definitions created for the TAIT (if any).
        let mut collected_lifetimes = Vec::new();
```